Classes/Hooks/AllowLanguageSynchronizationHook.php

```diff
@@ -4,10 +4,15 @@
 
 namespace WebVision\Deepltranslate\Core\Hooks;
 
+use Symfony\Component\DependencyInjection\Attribute\Autoconfigure;
 use TYPO3\CMS\Core\DataHandling\DataHandler;
 use TYPO3\CMS\Core\Utility\MathUtility;
 
-class AllowLanguageSynchronizationHook
+/**
+ * changes the l10nState with allowLanguageSynchronisation enabled to custom. THis is needed that DeepL can translate
```

Comment:
```suggestion
 * Changes the `l10nState` with `allowLanguageSynchronisation` enabled to custom. This is needed that DeepL can translate.
```

The wording is off .. what do you mean ? oO